packages/flutter_tools/templates/package/README.md.tmpl

```diff
@@ -1,14 +1,36 @@
-# {{projectName}}
+<!-- 
+This README describes the package. If you publish this package to pub.dev,
+then this README supplies most of the contents of the package page.
 
-{{description}}
+For information about how to write a good package README, see the guide for
+[writing package pages](https://dart.dev/guides/libraries/writing-package-pages). 
 
-## Getting Started
+For general information about developing packages, see the Dart guide for
+[creating packages](https://dart.dev/guides/libraries/create-library-packages)
+and the Flutter guide for
+[developing packages and plugins](https://flutter.dev/developing-packages). 
+-->
 
-This project is a starting point for a Dart
-[package](https://flutter.dev/developing-packages/),
-a library module containing code that can be shared easily across
-multiple Flutter or Dart projects.
+TODO: Put a short description here that helps potential users know whether
+this package might be useful for them.
 
-For help getting started with Flutter, view our 
-[online documentation](https://flutter.dev/docs), which offers tutorials, 
-samples, guidance on mobile development, and a full API reference.
+## Features
```

Comment:
I'd like to suggest we also add a section for "Support level". This is meant to describe if the package takes community contributions (PRs), and how responsive users should expect the publisher to be around issues.

Reply:
I think the `##Additional information` section implies it? 

We can add to the [writing package pages](https://dart.dev/guides/libraries/writing-package-pages) doc that authors are encouraged to add a support level.

Reply:
I guess I think of *Additional information* as miscellaneous content that is optional. I'm suggesting packages should be encouraged to always specify what their support level is.

Reply:
The _Additional Information_ section says “Tell users where to go next, how to leave feedback, etc.”

What about updating it into: “Tell users more about the package: where to find more information, how to contribute to the package, how to file issues, what response they can expect from the package authors, and more.”

If we want to _require_ authors to specify a support level, I think it’s better to make it a line in `pubspec.yaml`, so that it’s displayed in the metadata, instead of adding a separate section in the README.

Reply:
Any comment on the suggestion? @mit-mit

Reply:
I don't think we can add this to the pubspec, as we don't have any kind of structured format for this.

I think your text suggestion can work for now.

Reply:
Sounds good. Thank you!